e2e/components/radio/radio.e2e.ts

```diff
@@ -0,0 +1,33 @@
+describe('radio', function () {
+
+  describe('disabling behavior', function () {
+    beforeEach(function() {
+      browser.get('/radio');
+    });
+
+    it('should be checked when clicked', function () {
+      element(by.id('water')).click();
+      element(by.id('water')).getAttribute('class').then((value: string) => {
+        expect(value).toContain('md-radio-checked');
```

Comment:
We should also assert that the underlying `<input type="radio">` is checked as well (since it drives the accessibility of the component).